symbols/amenity/parking_entrance_multistorey.svg

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   width="14"
+   height="14"
+   version="1.1"
+   viewBox="0 0 14 14"
+   id="svg8">
+  <defs
+     id="defs12" />
+  <metadata
+     id="metadata2">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title></dc:title>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
```

Comment:
This svg file (`parking_entrance_multistorey.svg`) could be optimized by removing the unused stroke-width and fill rules, and the metadata. If you are using Inkscape, open the svg and then save as "optimized svg" format. Select the options to remove metadata and unused rules. This will create a much shorter file, like this:

```
<?xml version="1.0" encoding="UTF-8"?>
<svg width="14" height="14" version="1.1" viewBox="0 0 14 14" xmlns="http://www.w3.org/2000/svg">
 <path d="m0 0v10h2v-4h2.3807c3.5045-0.023785 3.5045-6 0-6zm2 2.0115v1.9885l2-2.255e-4c1.5622 2.255e-4 1.5728-1.9998 0-2.0115z" fill-rule="evenodd"/>
 <path d="m5 11h5v3l4-4-4-4v3h-5v2z"/>
</svg>
```

Reply:
Done :)

I made a confusion between simple svg and optimized svg